.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
How about something more specific, like "Let us know that the board agendas site is behaving differently than expected"?